### ISSUE TITLE
third web provider not recognized by react app

### ISSUE THREAD

**Issue by ahmedjsofficial** [thirdweb-dev/react#76]
**This is unknown error white adding this packages:**
`npm install @thirdweb-dev/react @thirdweb-dev/sdk ethers`
in react.js - react-18 .. my node version is node 18.2.0 ... when i installed these packages react js giving me error. The webpack is not compiled by node. The Stream in webpack node_modules is not defined. bla bla bla.. 

Plz tell me what's happing going on. ???
watting!
![WhatsApp Image 2022-06-02 at 11 34 04 PM](https://user-images.githubusercontent.com/90926687/171702491-7a52884b-2fee-41b4-92ff-c4ac508e3f1c.jpeg)
wating..

**Comment by jnsdls** [comment]
@jsstackdevelopers could you check if both your `@thirdweb-dev/sdk` and `@thirdweb-dev/react` versions are up to date?

If they are not please try to update those first and see if the error is still happening.

If it is still happening could you please provide a small reproduction case? Code sandbox, github repo, replit... something that will let us debug this, thanks!

**Comment by jsstackdevelopers** [comment]
Yes @jnsdls its updated. Every thing is updated but why its giving me errors i don't know.

**Comment by jnsdls** [comment]
> Yes @jnsdls its updated. Every thing is updated but why its giving me errors i don't know.

Please tell me what you're using to build your app. Is this based on `create-react-app`? Is it based on `vite`? Is it your own webpack-setup? How are you building your app, if you tell me that I should be able to reproduce it and fix it.
